src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/client/fakeplayer/CommandFakePlayer.kt

```diff
@@ -0,0 +1,362 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.features.command.commands.client.fakeplayer
+
+import com.mojang.authlib.GameProfile
+import net.ccbluex.liquidbounce.event.Listenable
+import net.ccbluex.liquidbounce.event.events.GameTickEvent
+import net.ccbluex.liquidbounce.event.events.NotificationEvent
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.command.Command
+import net.ccbluex.liquidbounce.features.command.CommandException
+import net.ccbluex.liquidbounce.features.command.builder.CommandBuilder
+import net.ccbluex.liquidbounce.features.command.builder.ParameterBuilder
+import net.ccbluex.liquidbounce.lang.translation
+import net.ccbluex.liquidbounce.utils.client.*
+import net.minecraft.entity.Entity
+import net.minecraft.network.packet.c2s.play.PlayerInteractEntityC2SPacket
+import net.minecraft.network.packet.s2c.play.ExplosionS2CPacket
+import net.minecraft.world.explosion.Explosion
+import net.minecraft.world.explosion.ExplosionBehavior
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.util.*
+
+/**
+ * Fake Player Command
+ *
+ * Allows you to spawn a client side player for testing purposes.
+ */
+object CommandFakePlayer : Listenable {
+
+    /**
+     * Stores all fake players.
+     */
+    private val fakePlayers: MutableList<FakePlayer> = ArrayList()
```

Comment:
You could just use `fakePlayers = ArrayList<FakePlayer>()` which would look nicer, but this is fine too.